src/main/java/org/apache/gossip/manager/random/RandomGossipManager.java

```diff
@@ -42,7 +43,14 @@ public static ManagerBuilder newBuilder() {
 
     private ManagerBuilder() {}
 
+    private void checkNotNull(Object arg, String msg) {
```

Comment:
Actually, based on @edwardcapriolo (which I just saw), [comment](https://issues.apache.org/jira/browse/GOSSIP-4?focusedCommentId=15315546&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-15315546), we should keep this, but throw an `IllegalArgumentException`.